modules/generator/processor/servicegraphs/servicegraphs.go

```diff
@@ -233,6 +244,15 @@ func (p *Processor) upsertDimensions(m map[string]string, resourceAttr []*v1_com
 	}
 }
 
+func (p *Processor) upsertPeerNode(e *store.Edge, spanAttr []*v1_common.KeyValue) {
+	for _, peerKey := range p.Cfg.PeerAttributes {
+		if v, ok := processor_util.FindAttributeValue(peerKey, spanAttr); ok {
+			e.PeerNode = v
```

Comment:
Looking at this logic - is it correct to say that the order of `PeerAttributes` is their priority?  If `net.sock.peer.addr` exists it will always be used first.  If so let's add a note to the documentation snippet in `service_graphs.md`

Reply:
Nice suggestions. Applied both.